src/components/ArticleView/Tools/PostCommentForm.jsx

```diff
@@ -0,0 +1,49 @@
+import { useParams } from "react-router-dom";
+import { postCommentByArticle_Id } from "../../../Utils/Api";
+import { useState } from "react";
+
+const PostCommentForm = ({comments, setComments}) => {
+  // set commentData state as an empty object
```

Comment:
Shouldn't need comments in our code, the code should read semantically from variable names and function names